defs/scenarios/storage/ceph/osd_slow_ops.yaml

```diff
@@ -0,0 +1,17 @@
+checks:
+  osd_slow_ops:
+    input:
+      path: var/log/ceph/ceph*.log
+    expr: '^([\d-]+)T([\d:]+\.\d)\S+ .+ slow ops, oldest one blocked .+'
+conclusions:
+  osd-slow-ops:
+    decision: osd_slow_ops
+    raises:
+      type: hotsos.core.issues.CephOSDError
+      message: >-
+        One or more Ceph OSDs has been experiencing slow ops.
+        Causes include network issues (could do connectivity checks via ping,
```

Comment:
Again you could check (some) network issues like packet drops or errors like in https://github.com/canonical/hotsos/blob/main/defs/scenarios/storage/ceph/mon_elections_flapping.yaml